extensions/hibernate-validator/runtime/src/main/java/io/quarkus/hibernate/validator/runtime/jaxrs/ResteasyViolationExceptionMapper.java

```diff
@@ -0,0 +1,112 @@
+package io.quarkus.hibernate.validator.runtime.jaxrs;
+
+import java.util.Iterator;
+import java.util.List;
+
+import javax.validation.ConstraintDeclarationException;
+import javax.validation.ConstraintDefinitionException;
+import javax.validation.GroupDefinitionException;
+import javax.validation.ValidationException;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.ResponseBuilder;
+import javax.ws.rs.core.Response.Status;
+import javax.ws.rs.ext.ExceptionMapper;
+import javax.ws.rs.ext.Provider;
+
+import org.jboss.resteasy.api.validation.ResteasyViolationException;
+import org.jboss.resteasy.api.validation.Validation;
+import org.jboss.resteasy.api.validation.ViolationReport;
+
+/**
+ *
+ * @author <a href="ron.sigal@jboss.com">Ron Sigal</a>
+ * @version $Revision: 1.1 $
+ *
+ *          Created Mar 31, 2012
+ */
```

Comment:
The exception mapper was moved from resteasy-core to resteasy-validator-provider, where it really belongs (actually, we had to do that because of a split package, afair). Considering you already had duplicated other resteasy validator provider classes, this looked like the most straightforward thing to do, instead of adding another dependency to quarkus hibernate validator, but we can do differently if you want.

Reply:
Fair enough for the javadoc comment, will remove it

Reply:
@asoldano looks good to me apart from this. Could you take care of it (by amending your initial commit)? I'll merge then.

Reply:
@gsmet done, thanks for the review.